### PR TITLE
Added python-minifier as a pacakge.

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/python-minifier/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  entry_points:
+    - pyminify=python_minifier.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python <3.11
+  run:
+    - python <3.11
```
Comment (ocefpaf) [inline recipes/python-minifier/meta.yaml:25]: Even if this works with py27 it is probably safer to add >=3.6,<3.11.

Comment (thewchan) [reply recipes/python-minifier/meta.yaml]: Got it I repushed with this addition.

File: recipes/python-minifier/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  noarch: python
+  entry_points:
+    - pyminify=python_minifier.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.11
```
Comment (dopplershift) [inline recipes/python-minifier/meta.yaml:23]: Upstream says `setuptools_scm` as well:
```suggestion
    - python >=3.6,<3.11
    - setuptools_scm
```

Comment (thewchan) [reply recipes/python-minifier/meta.yaml:23]: Roger that

**Comment by thewchan** [comment]
@dopplershift I think this is good to go?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**